app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -30,7 +34,13 @@ class DragAndDropSortInteractionViewModel(
     val item = choiceItems[indexFrom]
     choiceItems.removeAt(indexFrom)
     choiceItems.add(indexTo, item)
-    adapter.notifyItemMoved(indexFrom, indexTo)
+    if (allowMultipleItemsInSamePosition) {
+      choiceItems[indexFrom].itemIndex = indexFrom
+      choiceItems[indexTo].itemIndex = indexTo
+      (adapter as BindableAdapter<*>).setDataUnchecked(choiceItems)
+    } else {
+      (adapter as BindableAdapter<*>).notifyItemMoved(indexFrom, indexTo)
```

Comment:
Why not also have this for the case of dragging groups?

Reply:
because if we call notifyItemMoved it does not update the content of items which is in our case is the icon of merging

Reply:
Could you also use notifyItemChanged()?

Reply:
I did try that out as well but it causes weird effects